### ISSUE TITLE
UnitOfMeasurement missing IN from possible values.

### ISSUE THREAD

**Issue by cpiggott** [jlevers/selling-partner-api#119]
InvalidArgumentException: Invalid value for enum '\SellingPartnerApi\Model\FbaInbound\UnitOfMeasurement', must be one of: 'inches', 'centimeters' in file /home/vagrant/code/vendor/jlevers/selling-partner-api/lib/ObjectSerializer.php on line 345

It seems that the only valid values are "inches" and "centimeters" but the API returns "IN" for the measurement. Adding the "IN" to the UnitOfMeasurement class fixes this, but I didn't know if this was a generated class or something else.

The body that is returned from the Amazon API is the following:
```
{

	"payload": {
		"TransportContent": {
			"TransportHeader": {
				"SellerId": "XXXXXX",
				"ShipmentId": "XXXXXXX",
				"IsPartnered": true,
				"ShipmentType": "SP"
			},
			"TransportDetails": {
				"PartneredSmallParcelData": {
					"PackageList": [{
							"Dimensions": {
								"Length": 21.0,
								"Width": 17.0,
								"Height": 17.0,
								"Unit": "IN"
							},
							"Weight": {
								"Value": 41.0,
								"Unit": "pounds"
							},
							"CarrierName": "UNITED_PARCEL_SERVICE_INC",
							"TrackingId": "XXXXXXXXXXXXX",
							"PackageStatus": "IN_TRANSIT"
						}, {
							"Dimensions": {
								"Length": 21.0,
								"Width": 17.0,
								"Height": 17.0,
								"Unit": "IN"
							},
							"Weight": {
								"Value": 41.0
								"Unit": "pounds"
							},
							"CarrierName": "UNITED_PARCEL_SERVICE_INC",
							"TrackingId": "XXXXXXXX",
							"PackageStatus": "SHIPPED"
						}],
					"PartneredEstimate": {
						"Amount": {
							"CurrencyCode": "USD",
							"Value": 21.11
						},
						"VoidDeadline": "2021-10-15T00:56:03Z"
					}
				}
			}
			,
			"TransportResult": {
				"TransportStatus": "CONFIRMED"
			}
		}
	}
}
```

**Comment by jlevers** [comment]
Thanks for reporting -- yep, this can be fixed in the models. I'll comment and close this issue when I push a release with an update that fixes this (should be quite soon).

**Comment by jlevers** [comment]
Fixed in `v4.1.1`.
